### PR TITLE
Booking contact details

### LINKED ISSUE #207: Add contacts information related to booking
Add phone no. & email on Single Booking page - data already available in GraphQL

screen shot 2018-05-23 at 15 10 25

"Write us a message" should be just simple "mailto:..." link

Zeplin: https://zpl.io/adAW4xe

### PR DESCRIPTION
**Pull request by dinodsaurus**
closes #207 <br/><br/><br/><url>LiveURL: https://smartfaq-207-booking-contact-information.surge.sh</url>
</url>

### CODE REVIEW

File: src/SingleBookingPage/bookingItem/Contact.js [hunk 1]
```diff
@@ -1,97 +1,117 @@
...
-          <Phone customColor="#00a991" />
-        </div>
-        <Typography size="normal" type="active">
-          +42 987 876 567 CZ
+type Props = {|
+  contactDetails: {|
+    phone: string,
+    email: string,
+  |},
+|};
```
Comment (dinodsaurus) [inline src/SingleBookingPage/bookingItem/Contact.js]: Is there a better way on doing this?

Comment (dinodsaurus) [reply src/SingleBookingPage/bookingItem/Contact.js]: tnx

File: src/SingleBookingPage/BookingDetail.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ class BookingDetail extends React.Component<Props> {
             <button className="manage-booking">Manage my booking</button>
           </a>
         </div>
-        {isUrgent && <Contact />}
+        {isUrgent && <Contact booking={booking} />}
```
Comment (jaroslav-kubicek) [inline src/SingleBookingPage/BookingDetail.js]: I would suggest to use different name for prop `<Contact booking={booking} />` looks suspicious at the first sight.

Comment (jaroslav-kubicek) [reply src/SingleBookingPage/BookingDetail.js]: e.g. info or even default data

or you could create fragment on `BookingContactDetails` and then `<Contact contacts={booking.contactDetails} />`

Comment (dinodsaurus) [reply src/SingleBookingPage/BookingDetail.js]: Thanks for this suggestion, I changed the prop name to `info`, but im not sure if I get the second part of you suggestion. If you could make a quick example that would be great 🤔

**Pull request merged**